web/themes/custom/jcc_elevated/templates/paragraphs/paragraph--steps.html.twig

```diff
@@ -0,0 +1,47 @@
+{{ attach_library("jcc_storybook/Steps") }}
+{{ attach_library("jcc_storybook/Container") }}
```

Comment:
Please remove "Container" and "Stack" for the reasons mentioned previously.

Reply:
Steps template has been completely redone.